src/siaskydev.test.ts

```diff
@@ -29,4 +31,48 @@ describe.skip("siasky.dev end to end", () => {
     const buf = Buffer.from(parts[1], "base64");
     expect(buf.toString("ascii")).toEqual("thisistext");
   });
+
+  it("support rev number update higher than 255", async () => {
+    // create a random user
+    const rand = randomNumber(0, 1e6);
+    const user = new User(`john.doe+${rand}@gmail.com`, "test1234");
+    console.log(`Uploading for userid ${user.id}`);
+
+    // verify there is no entry
+    let existing = null;
+    try {
+      existing = await promiseTimeout(client.lookupRegistry(user, fileID), 1e4);
+    } catch (error) {
+      // expected
+    }
+    expect(existing).toBeNull();
+
+    // build the registry value
+    let revision = 0;
+    const skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg";
+
+    // update the registry 256 times
+    for (revision; revision <= 260; revision += 10) {
+      if (revision && revision % 50 === 0) {
+        console.log(`Iteration ${revision}...`);
+      }
+
+      // calculate the registry value and sign it
+      const value: RegistryValue = {
+        tweak: HashFileID(fileID),
+        data: skylink,
+        revision,
+      };
+      const signature = user.sign({ message: HashRegistryValue(value) });
+
+      // update the registry entry
+      const updated = await client.updateRegistry(user, fileID, { value, signature });
+      expect(updated).toBe(true);
+    }
+
+    // verify the revision in the registry
+    existing = await client.lookupRegistry(user, fileID);
+    expect(existing.value.revision).toBeGreaterThan(255);
```

Comment:
```suggestion
    const entry = await client.lookupRegistry(user, fileID);
    expect(entry.value.revision).toBeGreaterThan(255);
```

don't overwrite a variable if it's not necessary